docs/LearningOutcomes.adoc

```diff
@@ -7,6 +7,7 @@
 ifdef::env-github[]
 :note-caption: :information_source:
 endif::[]
+:repoURL: https://github.com/se-edu/addressbook-level1
```

Comment:
Seems like `build.gradle` already declare a similar attribute, so no need to declare another one here again. Will use `build.gradle`'s `site-githuburl` instead of declaring our own `repoURL`.

https://github.com/se-edu/addressbook-level1/blob/a9270ee306badf96936da1ba6dbf98f95cc49ed7/build.gradle#L28-L30

Reply:
Of course, the URL in `build.gradle` still needs to be manually changed when forking the repository.

Reply:
Are we supposed to do that already? :-o I didn't do that when we forked repos this time, and it is a bit of a hassle too, not to mention the additional work needed to keep the forks synced with upstream?

Reply:
It's only used when publishing the website.